pkgs/applications/networking/p2p/opentracker/default.nix

```
@@ -11,6 +11,12 @@ stdenv.mkDerivation {
 
   buildInputs = [ libowfat zlib ];
 
+  postPatch = ''
+    substituteInPlace Makefile \
+      --replace 'LIBOWFAT_HEADERS=$(PREFIX)/libowfat' 'LIBOWFAT_HEADERS=${libowfat}/include/libowfat' \
+      --replace 'LIBOWFAT_LIBRARY=$(PREFIX)/libowfat' 'LIBOWFAT_LIBRARY=${libowfat}/lib'
+  '';
```

Does this work as well?

```suggestion
  makeFlags = [
    "LIBOWFAT_HEADERS=${libowfat}/include/libowfat"
    "LIBOWFAT_LIBRARY=${libowfat}/lib"
  ];
```

---

Did you forgot to push or to answer?

---

I had written an answer, but hit "resolve" instead of "comment".

Answer: No, this wouldn't work. The Makefile does not respect environment variables (no `?=` in var assignment), so we would need to patch the Makefile anyway.

---

makeFlags are not passed environment variables but via argv and should override `=` assignments.

---

I stand corrected. PR updated.